environments/development/base/outputs.tf

```
@@ -0,0 +1,19 @@
+output "region" {
+  description = "AWS region"
+  value       = var.region
+}
+
+output "public_subnet_id" {
+  description = "subnet id"
+  value       = module.vpc.public_subnets[*]
+}
```

I think the output here is [already a list](https://github.com/trade-tariff/terraform-aws-vpc/blob/master/outputs.tf#L110), so we don't need to use a splat.

```suggestion
output "public_subnet_ids" {
  description = "List of public subnet IDs"
  value       = module.vpc.public_subnets
}
```